crates/holochain/src/conductor/conductor/hc_p2p_handler_impl.rs

```diff
@@ -10,7 +23,12 @@ impl holochain_p2p::event::HcP2pHandler for Conductor {
         zome_call_params_serialized: ExternIO,
         signature: Signature,
     ) -> BoxFut<'_, HolochainP2pResult<SerializedBytes>> {
-        Box::pin(async { unimplemented!() })
+        Box::pin(async {
+            self.cell_by_parts(&dna_hash, &to_agent)
+                .await?
+                .handle_call_remote(dna_hash, to_agent, zome_call_params_serialized, signature)
+                .await
+        })
```

Comment:
Actually forward these calls to the correct cell... I'd forgotten to do this in my last PR.